ledger-tool/src/main.rs

```diff
@@ -1926,6 +1926,7 @@ fn main() {
                     );
                     assert!(bank.is_complete());
                     bank.squash();
+                    bank.force_flush_accounts_cache();
```

Comment:
how about calling `force_flush_accounts_cache()` in the `Bank::exhaustively_free_unused_resource` as well?

Reply:
unless long blockstore_processor run could starve the memory due to not flushing caches?

Reply:
Updated, to move `force_flush_accounts_cache()` from being called outside (previously called a few lines above) to inside `exhaustively_free_unused_resource`.